$OpenBSD$
Defining _POSIX_C_SOURCE sets __BSD_VISIBLE to 0, resulting in some interfaces
like pipe2() or dup3() unavailable.
--- src/corelib/tools/qelapsedtimer_unix.cpp.orig	Sun Jul 10 20:00:54 2016
+++ src/corelib/tools/qelapsedtimer_unix.cpp	Sun Jul 10 20:01:30 2016
@@ -32,7 +32,9 @@
 ****************************************************************************/
 
 // ask for the latest POSIX, just in case
-#define _POSIX_C_SOURCE 200809L
+#ifndef __OpenBSD__
+# define _POSIX_C_SOURCE 200809L
+#endif
 
 #include "qelapsedtimer.h"
 #if defined(Q_OS_VXWORKS)
